src/compiler/diagnosticMessages.json

```diff
@@ -1574,7 +1574,8 @@
     },
     "Index expression arguments in 'const' enums must be of type 'string'.": {

```

Comment:
Change this to say, "A const enum member can only be accessed using a string literal."